stock_move_backdating/__openerp__.py

```diff
@@ -19,23 +19,19 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+
+
 {
-    "name": "Stock Move Backdating",
-    "version": "1.0",
-    'author': ['Marco Dieckhoff, BREMSKERL', 'Agile Business Group'],
-    "category": "Stock Logistics",
+    'name': 'Stock Move Backdating',
+    'version': '8.0.1.0.0',
+    'category': 'Stock Logistics',
+    'author': 'Marco Dieckhoff, BREMSKERL, Agile Business Group,'
+    ' Odoo Community Association (OCA)',
     'website': 'www.bremskerl.com',
-    'license': 'AGPL-3',
-    "depends": ["stock"],
-    "summary": "Allows back-dating of stock moves",
-    "description": """This module allows to register old stock moves
-    (with date != now).
-    On stock moves, user can specify the "Actual Movement Date", that will be
-    used as movement date""",
+    'depends': ['stock_account'],
```

Comment:
It says Pedro , my mistake , the change need to import both stock that account , with one module useful to have all dependencies met , and then have the link with the billing . Sorry but I wrote the module 4 months ago ...